hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionPolicy.java

```diff
@@ -44,6 +46,14 @@
     @Max(5000)
     private int sendingDelay = DEFAULT_SENDING_DELAY;
 
+    @Min(1)
+    @Max(10)
+    private double backoffMultiplier = DEFAULT_BACKOFF_MULTIPLIER;
+
+    @Min(1)
+    @Max(600)
+    private int backoffMaxInterval = DEFAULT_BACKOFF_MAX_INTERVAL;
```

Comment:
In Hermes we have this unwritten rule that we add a unit of time to a variable name if the unit is different than milliseconds. Therefore please consider adding here a suffix `InSec` to the variable name.